erpnext/assets/doctype/asset_repair/asset_repair.py

```diff
@@ -54,10 +57,13 @@ class AssetRepair(AccountsController):
 	def validate(self):
 		self.asset_doc = frappe.get_doc("Asset", self.asset)
 		self.validate_dates()
+		self.validate_purchase_invoice_repair_cost()
```

Comment:
Add another validation to validate the expense account, ensuring the selected expense account is present in the Purchase Invoice.